crates/language/src/language_registry.rs

```diff
@@ -57,6 +57,18 @@ impl LanguageName {
     }
 }
 
+impl Into<SharedString> for LanguageName {
```

Comment:
Would it be more idiomatic to implement `From<LanguageName> for SharedString` instead?

Reply:
Yes, nice catch! Changed in 0e3a1ba722e292f971eb5f30e03c76008b62ea4d